src/Subscriptions/SubscriptionRouter.php

```diff
@@ -23,4 +23,15 @@ public function pusher($router): void
             'uses' => SubscriptionController::class.'@webhook',
         ]);
     }
+
+    /**
+     * @param   \Illuminate\Routing\Router  $router
```

Comment:
```suggestion
     * @param  \Illuminate\Routing\Router  $router
```

Reply:
I just kept it the same with pusher($routes).

Reply:
I guess it might be because of Lumen, in which case there actually is no shared router interface. The PHPDoc should be `\Illuminate\Contracts\Routing\Registrar|\Laravel\Lumen\Routing\Router` in that case.